README.md

```diff
@@ -160,9 +193,14 @@ described below to verify that the code still works. All of the following comman
 directory.
 
 ### Unit tests
+To run unit tests within genet python environment:
 
     python -m pytest -vv tests
 
+and within a docker container:
+
+    docker run cml-genet python -m pytest -vv tests
```

Comment:
does `docker run cml-genet pytest -vv` have the same effect?

Reply:
`docker run cml-genet pytest -vv tests` does, will change